lib/ldclient-rb/ldclient.rb

```diff
@@ -247,8 +247,19 @@ def match_target?(target, user)
 
     end
 
+    def match_user?(variation, user)
+      if !!variation[:userTarget]
+        return match_target?(variation[:userTarget], user)
+      end
+      return false
+    end
+
     def match_variation?(variation, user)
```

Comment:
Yeah, some of the names are a bit unfortunate, but I want to keep the SDKs as consistent as I can. It makes it easier to port code between them. So I'd rather not have to re-name this everywhere.